commit 75994c2981d7df79f8f235e44058f1c127265cf4 (refs/changes/71/5571/3)
Author: Kody A Kantor <kody@kkantor.com>
Date:   2019-02-14T15:24:54+00:00 (8 months ago)
    
    OS-7573 smartos zfstest runfile should omit unreliable tests
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/test/zfs-tests/runfiles/smartos.run b/usr/src/test/zfs-tests/runfiles/smartos.run
index c0a2dbce9e..952be3652f 100644
--- a/usr/src/test/zfs-tests/runfiles/smartos.run
+++ b/usr/src/test/zfs-tests/runfiles/smartos.run
@@ -45,8 +45,7 @@ tests = ['zpool_offline_001_pos', 'zpool_offline_002_neg']
 tests = ['zpool_online_001_pos', 'zpool_online_002_neg']
 
 [/opt/zfs-tests/tests/functional/cli_root/zpool_remove]
-tests = ['zpool_remove_001_neg', 'zpool_remove_002_pos',
-    'zpool_remove_003_pos']
+tests = ['zpool_remove_002_pos', 'zpool_remove_003_pos']
 
 [/opt/zfs-tests/tests/functional/cli_root/zpool_replace]
 tests = ['zpool_replace_001_neg', 'zpool_replace_002_neg']
@@ -119,11 +118,6 @@ tests = ['async_destroy_001_pos']
 [/opt/zfs-tests/tests/functional/grow_pool]
 tests = ['grow_pool_001_pos']
 
-[/opt/zfs-tests/tests/functional/grow_replicas]
-tests = ['grow_replicas_001_pos']
-pre =
-post =
-
 [/opt/zfs-tests/tests/functional/history]
 tests = ['history_001_pos', 'history_002_pos', 'history_003_pos',
     'history_004_pos', 'history_005_neg', 'history_006_neg',
@@ -132,10 +126,6 @@ tests = ['history_001_pos', 'history_002_pos', 'history_003_pos',
 [/opt/zfs-tests/tests/functional/holes]
 tests = ['holes_sanity']
 
-[/opt/zfs-tests/tests/functional/inheritance]
-tests = ['inherit_001_pos']
-pre =
-
 [/opt/zfs-tests/tests/functional/inuse]
 tests = ['inuse_001_pos', 'inuse_003_pos', 'inuse_004_pos',
     'inuse_005_pos', 'inuse_006_pos', 'inuse_007_pos', 'inuse_008_pos',
@@ -197,10 +187,6 @@ tests = ['poolversion_001_pos', 'poolversion_002_pos']
 tests = ['quota_001_pos', 'quota_002_pos', 'quota_003_pos', 'quota_004_pos',
     'quota_005_pos', 'quota_006_neg']
 
-[/opt/zfs-tests/tests/functional/redundancy]
-tests = ['redundancy_001_pos', 'redundancy_003_pos',
-    'redundancy_004_neg']
-
 [/opt/zfs-tests/tests/functional/refquota]
 tests = ['refquota_001_pos', 'refquota_002_pos', 'refquota_003_pos',
     'refquota_004_pos', 'refquota_005_pos', 'refquota_006_neg']
@@ -220,13 +206,12 @@ tests = ['reservation_001_pos', 'reservation_002_pos', 'reservation_003_pos',
     'reservation_004_pos', 'reservation_005_pos', 'reservation_006_pos',
     'reservation_007_pos', 'reservation_008_pos', 'reservation_009_pos',
     'reservation_010_pos', 'reservation_011_pos', 'reservation_012_pos',
-    'reservation_013_pos', 'reservation_014_pos', 'reservation_015_pos',
-    'reservation_016_pos', 'reservation_017_pos', 'reservation_018_pos',
-    'reservation_019_pos', 'reservation_020_pos', 'reservation_021_neg',
-    'reservation_022_pos']
+    'reservation_014_pos', 'reservation_015_pos', 'reservation_016_pos',
+    'reservation_018_pos', 'reservation_019_pos', 'reservation_020_pos',
+    'reservation_021_neg', 'reservation_022_pos']
 
 [/opt/zfs-tests/tests/functional/rootpool]
-tests = ['rootpool_002_neg', 'rootpool_003_neg']
+tests = ['rootpool_003_neg']
 
 [/opt/zfs-tests/tests/functional/rsend]
 tests = ['rsend_001_pos', 'rsend_002_pos', 'rsend_003_pos', 'rsend_006_pos',
@@ -237,10 +222,6 @@ tests = ['rsend_001_pos', 'rsend_002_pos', 'rsend_003_pos', 'rsend_006_pos',
     'send-c_incremental',
     'send-c_recv_dedup']
 
-[/opt/zfs-tests/tests/functional/scrub_mirror]
-tests = ['scrub_mirror_001_pos', 'scrub_mirror_002_pos',
-    'scrub_mirror_003_pos', 'scrub_mirror_004_pos']
-
 [/opt/zfs-tests/tests/functional/slog]
 tests = ['slog_001_pos', 'slog_002_pos', 'slog_003_pos', 'slog_004_pos',
     'slog_005_pos', 'slog_006_pos', 'slog_007_pos', 'slog_008_neg',
@@ -293,10 +274,6 @@ tests = ['zvol_dump_raidz']
 pre =
 post =
 
-[/opt/zfs-tests/tests/functional/zvol/zvol_misc]
-tests = ['zvol_misc_001_neg', 'zvol_misc_002_pos', 'zvol_misc_003_neg',
-    'zvol_misc_004_pos', 'zvol_misc_005_neg', 'zvol_misc_006_pos']
-
 [/opt/zfs-tests/tests/functional/libzfs]
 tests = ['many_fds']
 pre =
